vue/components/ui/molecules/select-checkboxes/select-checkboxes.vue

```diff
@@ -170,6 +177,45 @@ export const SelectCheckboxes = {
             },
             deep: true
         }
+    },
+    methods: {
+        onKey(key) {

```

Comment:
This seems to be [select-ripe `onKey` logic ](https://github.com/ripe-tech/ripe-components-vue/blob/a2b9d2aa0e199d59211232f6cf15d950ef052206/vue/components/ui/molecules/select/select.vue#L369) I would call the method something like `keyDebounce` and add this to a mixin (same for other logic you think it can be used in a general context), and share between the two components, or probably better yet, just have the select-ripe sending the key event and the `keyBuffer` with the `on:key` event and use that

Reply:
That makes total sense, adding that in!